### PR TITLE
Tests for authentication service

### PR DESCRIPTION
**Pull request by kornys**

### CODE REVIEW

File: systemtests/src/main/java/io/enmasse/systemtest/AdminResourcesManager.java [hunk 1]
```diff
@@ -6,43 +6,53 @@
...
+            for (AddressSpacePlan addressSpacePlan : addressSpacePlans) {
+                adminApiClient.deleteAddressSpacePlan(addressSpacePlan);
+            }
 
-        for (AddressPlan addressPlan : addressPlans) {
-            adminApiClient.deleteAddressPlan(addressPlan);
-        }
+            for (AddressPlan addressPlan : addressPlans) {
+                adminApiClient.deleteAddressPlan(addressPlan);
+            }
```
[the remark this answers is not in the document]
Comment (kornys) [reply systemtests/src/main/java/io/enmasse/systemtest/AdminResourcesManager.java]: not sure, but it was autoformatted by Idea

**Comment by kornys** [comment]
@lulf test commom.api.AuthServiceTest#testAuthServiceWithoutDeletingClaim does not work, so my question is if we should disable it, or make fix in #2558

**Comment by lulf** [comment]
@kornys Disable it for now

**Comment by kornys** [comment]
@enmasse-ci run tests testcase=common.api.AuthServiceTest

**Comment by kornys** [comment]
@enmasse-ci run tests testcase=common.api.AuthServiceTest

**Pull request merged**